js/ui/form/components/field_item.js

```diff
@@ -28,6 +28,7 @@ import { renderLabel } from './label';
 const TEMPLATE_WRAPPER_CLASS = 'dx-template-wrapper';
 const VALIDATION_TARGET_CLASS = 'dx-validation-target';
 const INVALID_CLASS = 'dx-invalid';
+const SWITCH_GROUP_SPACING_CLASS = 'dx-swithc-group-spacing';
```

Comment:
I think the name of this class is bad. Get assistance from Vlada if you use difficulties

Reply:
So I did, now we are discussing, the name 'toogle' part

Reply:
Tech writers suggested an option dx-toggle-controls-paddings